custom_components.json

```diff
@@ -0,0 +1,14 @@
+{
+    "philips_android_tv": {
+        "version": "0.0.2",
+        "local_location": "/custom_components/philips_android_tv/__init__.py",
+        "remote_location": "https://raw.githubusercontent.com/olbjan/ha_philips_2016/master/philips_android_tv/__init__.py",
```

Comment:
Probably should change to nstrelow.

Better would be having it as an official component, but more work is needed for that.